lms/static/styles/components/_ValidationMessage.scss

```diff
@@ -37,6 +40,7 @@
   &:hover {
     cursor: pointer;
   }
+  @include mixins.input-focus;
```

Comment:
I generally prefer to put mixins at the top of a CSS rule because that makes it easier to reason about the final values of properties, as it avoids a mixin overriding a property defined specifically for a CSS rule which is almost always going to be a mistake.